src/java/us/kbase/templates/perl_client.vm.properties

```diff
@@ -572,69 +677,66 @@ sub call {
 
 #if( $enable_client_retry )
 
-    my @retries = (1, 2, 5, 10, 20, 60, 60, 60, 60, 60, 60);
-    my %codes_to_retry =  map { $_ => 1 } qw(110 408 502 503 504 200) ;
+    my @retries = ( 1, 2, 5, 10, 20, 60, 60, 60, 60, 60, 60 );
+    my %codes_to_retry =  map { $_ => 1 } qw( 110 408 502 503 504 200 ) ;
     my $n_retries;
 
     while (1)
 #end
 
     {
-	if ($uri =~ /\?/) {
-	    $result = $self->_get($uri);
-	}
-	else {
-	    Carp::croak "not hashref." unless (ref $obj eq 'HASH');
-	    $result = $self->_post($uri, $headers, $obj);
-	}
-#if( $enable_client_retry )
+        if ($uri =~ /\?/) {
+            $result = $self->_get($uri);
+        }
+        else {
+            Carp::croak "not hashref." unless (ref $obj eq 'HASH');
+            $result = $self->_post($uri, $headers, $obj);
+        }
+        #if( $enable_client_retry )
+
+        #
+        # Bail early on success.
+        #
+        if ($result->is_success) {
+            if ($n_retries) {
+                print STDERR strftime("%F %T", localtime), ": Request succeeded after $n_retries retries\n";
+            }
+            last;
+        }
+        $n_retries++;
+
+        #
+        # Failure. See if we need to retry and loop, or bail with
+        # a permanent failure.
+        #
 
-	#
-	# Bail early on success.
-	#
-	if ($result->is_success)
-	{
-	    if ($n_retries)
-	    {
-		print STDERR strftime("%F %T", localtime), ": Request succeeded after $n_retries retries\n";
-	    }
-	    last;
-	}
-	$n_retries++;
-
-	#
-	# Failure. See if we need to retry and loop, or bail with
-	# a permanent failure.
-	#
-	
         my $code = $result->code;
-	my $msg = $result->message;
-	my $want_retry = 0;
-	if ($codes_to_retry${empty_escaper}{$code})
-	{
-	    $want_retry = 1;
-	}
-	elsif ($code eq 500 && defined( $result->header('client-warning') )
-	       && $result->header('client-warning') eq 'Internal response')
-	{
-	    #
-	    # Handle errors that were not thrown by the web
-	    # server but rather picked up by the client library.
-	    #
-	    # If we got a client timeout or connection refused, let us retry.
-	    #
-	    
-	    if ($msg =~ /timeout|connection refused/i)
-	    {
-		$want_retry = 1;
-	    }
-	    
-	}
-	
+        my $msg = $result->message;
+        my $want_retry = 0;
+        if ($codes_to_retry${empty_escaper}{$code})
+        {
```

Comment:
The perltidy linter didn't catch this, probably because it's got some template variables in the line (the `$codes_to_retry${empty_escaper}{$code}`). I have gone over a fair amount of this code by hand, so it's likely to be fixed in a future PR.